Issue 8 install deps

tdfranklin opened the pull request:
Unfortunately, it looks like this PR is showing all of the changes you made as well as the changes from my other PR.  I'm sorry about that, we'll get these cleaned up.

In any case, the only thing that was added here are the changes on core/cli/cli.js from lines 178 to the end.  It's just the issues we were discussing in issue #8 .

brillout commented on core/cli/cli.js:
```diff
@@ -118,3 +152,45 @@ function green_checkmark() {
+
+    child.stdout.on('data', data => {
+        console.log(data);
+    }
```
A closing `)` is missing

brillout commented on core/cli/cli.js:
```diff
@@ -118,3 +152,45 @@ function green_checkmark() {
+
+    child.on('close', code => {
+        console.log(`process completed with code: ${code}`);
+    }
```
A closing `)` is missing here as well

brillout commented on core/cli/cli.js:
```diff
@@ -118,3 +152,45 @@ function green_checkmark() {
+    }
+}
+
+process.on('unhandledRejection', err => {
```
I'd put this at the beginning of the file: We want to listen to rejected promises as soon as the program starts

brillout commented:
Other than the 2 missing `)` it works now 👍 

Don't worry about the commits order, I'll merge and take care of it

tdfranklin commented:
So sorry about the missing `)`... Not sure how I missed those.  Probably was just too tired last night, lol.  In any case, fixed that and moved the listener to the top.
